File: Support-EJ1/Captcha/WebApplication1/WebApplication1/Controllers/HomeController.vb
Imports Syncfusion.JavaScript
Public Class HomeController
    Inherits System.Web.Mvc.Controller
    Function Index() As ActionResult
        Dim targetInstance As New Syncfusion.JavaScript.Models.CaptchaProperties()
        Dim strangeType As Type = GetType(Syncfusion.JavaScript.Models.CaptchaProperties)
        Dim PropertyUppercase As Reflection.PropertyInfo = strangeType.GetProperty("EnableAudio")
        PropertyUppercase.SetValue(targetInstance, True)
        PropertyUppercase = strangeType.GetProperty("EnableRefreshImage")
        PropertyUppercase.SetValue(targetInstance, True)
        PropertyUppercase = strangeType.GetProperty("EnableCaseSensitivity")
        PropertyUppercase.SetValue(targetInstance, True)
        PropertyUppercase = strangeType.GetProperty("Height")
        PropertyUppercase.SetValue(targetInstance, 66)

        ViewBag.CaptchaUrl = targetInstance.RenderCaptchaImage
        PropertyUppercase = strangeType.GetProperty("EncryptedCode")
        ViewBag.EncryptedCode = PropertyUppercase.GetValue(targetInstance)
        Return View()
    End Function

    Function About() As ActionResult
        ViewData("Message") = "Your application description page."

        Return View()
    End Function

    Function Contact() As ActionResult
        ViewData("Message") = "Your contact page."

        Return View()
    End Function

    Function Refresh(params As CaptchaParams) As ActionResult
        Return params.CaptchaActions
    End Function


End Class

File: EJ1-Components-exmples/DateTimePicker/MVC/DateTimePickerFor/HomeController.cs
using Syncfusion.JavaScript.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace DateTimePicker_Helper.Controllers
{
    public class HomeController : Controller
    {
        DateModel model = new DateModel();
        public ActionResult Index()
        {
            model.Value = DateTime.Now;
            model.number = 66;
            model.percent = 75;

            model.currency = 89;
            return View(model);
        }

        [HttpPost]
        public ActionResult Index(DateModel model)
        {
            
            return View(model);

        }

        public ActionResult About()
        {
            ViewBag.Message = "Your application description page.";

            return View();
        }

        public ActionResult Contact()
        {
            ViewBag.Message = "Your contact page.";

            return View();
        }
    }

    public class DateModel
    {
        public DateTime Value { get; set; }

        public int number { get; set; }

        public int percent { get; set; }

        public int currency { get; set; }
    }
}